server/translate.js

```diff
@@ -0,0 +1,19 @@
+const Translate = require('@google-cloud/translate');
+const projectId = 'missionhack-198303'
+const translator = new Translate({
+	projectId: projectId
+});
+
+function translateMsg(msg, lang) {
+	let translation = 'unable to translate';
+
+	return translator.translate(msg, lang)
+	.then(results => {
```

Comment:
This can be simplified to a one-liner using ES6 stuff

.then(results => results)

Reply:
Can we return results[0] here?